tests/integration/_cases/organizations/organizations-no-subcommand.trycmd

```diff
@@ -0,0 +1,25 @@
+```
+$ sentry-cli organizations
+? failed
```

Comment:
Not ideal that we just write `failed` - shouldn’t we leave a message about why we have an error (missing required sub command?)

Reply:
We kinda do. It will print the help page with usage example. This is how clap implements it, and we do the same for every command. `failed` here just means non-0 exit code.